apps/demos/Demos/Chat/AIAndChatbotIntegration/description.md

```diff
@@ -1 +1,18 @@
-Chat is an interactive interface that allows users to send and receive messages in real time.
+This demo uses an AI service and the DevExtreme Chat component to create a chatbot UI.
+
+## Handling dataSource (reloadOnChange: false)
+
+The Chat component's [dataSource](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#dataSource) is a [CustomStore](/Documentation/ApiReference/Data_Layer/CustomStore/) that implements its own load and insert functions. The Chat deactivates [reloadOnChange](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#reloadOnChange) to push updates directly into the store and update the conversation manually. See the [onMessageEntered](/Documentation/ApiReference/UI_Components/dxChat/Configuration/#onMessageEntered) event handler and the `processMessageSending` function to review the code that manages data transfer between the Chat and its data store.
```

Comment:
`into the store` Do you mean 'storage'?

Reply:
No, I mean the data store (an entity in DevExtreme data layer), like the CustomStore mentioned.

Reply:
Okay, thanks